src/packs/core_macros/Scan_3KPsYon0LoANWX54.yml

```diff
@@ -6,18 +6,16 @@ img: systems/lancer/assets/icons/macro-icons/tech_quick.svg
 scope: global
 command: |-
   let targets = Array.from(game.user.targets);
-
+  const actorPermissionLevel = -1;
```

Comment:
You may want to document this and possibly use CONST.DOCUMENT_PERMISSION_LEVELS to make it more clear.

Reply:
Also, perhaps it would be better to use `undefined` or `null` explicitly to sentinel the feature as turned off?

Reply:
I'll document it to make it more clear, sure. `null` can work fine. Values should be something like the following, right?
```
CONST.DOCUMENT_PERMISSION_LEVELS.NONE
CONST.DOCUMENT_PERMISSION_LEVELS.LIMITED
CONST.DOCUMENT_PERMISSION_LEVELS.OBSERVER
CONST.DOCUMENT_PERMISSION_LEVELS.OWNER
```

Reply:
Documented with a comment and used the suggested constants!